JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -1,12 +1,29 @@
 //
 //  JuiceMaker - FruitStore.swift
-//  Created by yagom. 
+//  Created by dasan & kyungmin.
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
-
-// 과일 저장소 타입
 class FruitStore {
+    private(set) var fruitInventory: [Int]
+
+    init(initialStock: Int = 10) {
+        fruitInventory = Array(repeating: initialStock,
+                               count: Fruit.allCases.count)
+    }
+    
+    func hasEnoughStock(fruit: Fruit, amount: Int) -> Bool {
+        if fruitInventory[fruit.inventoryIndex] >= amount {
+            return true
+        }
+        return false
+    }
```

Comment:
사람에 따라 다르겠습니다만, 저는 분기에 대한 결과는 같은 위계에 있는게 가독성이 좋다고 생각해서 컴파일러가 한번 더 읽더라도 다음과 같이 쓰는 편입니다.
```suggestion
    func hasEnoughStock(fruit: Fruit, amount: Int) -> Bool {
        if fruitInventory[fruit.inventoryIndex] >= amount {
            return true
        } else {
            return false
        }
    }
```